conf/application.conf

```diff
@@ -16,7 +16,7 @@ include "frontend.conf"
 
 appName="pension-scheme-return-frontend"
 
-play.http.router = testOnlyDoNotUseInAppConf.Routes
+play.http.router = prod.Routes
```

Comment:
We would like to keep using `test-only` routes defined in testOnlyDoNotUseInAppConf.routes. How should we do that? Would it be the opposite way around, i.e. adding testOnlyDoNotUseInAppConf.routes to prod.routes? FYI @westwater @cyeniceri

Reply:
Hey I'll explain the change in your team channel - there's connecting app config changes too.